transip/client.py

```diff
@@ -52,6 +53,20 @@ def convert_value(value):
 
     return value
 
+class SudsFilter(DocumentPlugin):
```

Comment:
```suggestion
class WSDLFixPlugin(DocumentPlugin):
```
Might be a better description, might other plugins be added?

Reply:
Agreed